docs/cmd/completion.md

```diff
@@ -33,6 +33,10 @@ To load completions for each session, execute once:
 ```bash
 containerlab completion zsh > "${fpath[1]}/_containerlab"
 ```
+> Note: `$fpath[1]` in this command refers to the first path in `$fpath`. Ensure you use
```

Comment:
can you change the blockquote style here to a theme supported info block that is enabled like this:

```
!!!info
    your text here
```